src/components/cart/cart.tsx

```diff
@@ -23,6 +23,7 @@ export const Cart = ({ className, initialIsOpen }: CartProps) => {
         if (initialIsOpen !== undefined) {
             setIsOpen(initialIsOpen);
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
please add comment which describes why we need to ignore changes to `initialIsOpen`, because I find it very complex to understand such cases.

Reply:
IMHO, the name of the property is self-defined, but added a comment

Reply:
if that value is only for initial state, doesn't that mean that it will never change ? 
if so, we don't need to ignore it